### DISCUSSION TITLE
Self-Hosted rest api can't be found

### DISCUSSION THREAD

**Discussion by trentguillory** [supabase/supabase#4750] [Questions]
Hi all - self hosting with docker went smoothly. I followed the simple setup to a T except for the docker version. `3.8` was reportedly invalid, so I changed it to `3.3`, as suggested by the CLI. Deployment went smoothly, studio works, the db works, but there's one thing: 

When I click on "API Docs" in the sidebar of Studio, I get a "500: Can't Connect" error. Has anyone seen this and resolved it?

Thanks!

_ps to Supabase team if you see this: love what you're doing ❤️_

**Comment by soedirgo** [comment]
Yup this is a known bug: https://github.com/supabase/supabase/issues/4740
